rskj-core/src/main/java/co/rsk/peg/federation/FederationTrackerImpl.java

```diff
@@ -0,0 +1,4 @@
+package co.rsk.peg.federation;
+
+public class FederationTrackerImpl extends FederationTracker<Federation> {
```

Comment:
There is no need for these two impl classes. You can leave `FederationTracker` as a normal class with the generics as it is.